README.md

```diff
@@ -547,6 +548,61 @@ instance.appendAppSubversion(8888);
 instance.connect();
 ```
 
+## <a id="getattributioninfo"></a>getAttributionInfo method
+
+Description on how to use getAttributionInfo method.
+
+```java
+import com.tenjin.android.TenjinSDK;
+import com.tenjin.android.Callback;
+
+public class TenjinDemo extends ActionBarActivity {
+
+    //...other callbacks are here
+
+    @Override
+    public void onResume() {
+        //standard code
+        super.onResume()
+
+        //Integrate TenjinSDK connect call
+        String apiKey = "<API_KEY>";
+        TenjinSDK instance = TenjinSDK.getInstance(this, apiKey);
+        instance.connect();
+
+        instance.getAttributionInfo(new Callback() {
+            @Override
+            public void onSuccess(Map<String, String> data) {
+                if (data.containsKey(TenjinConsts.ATTR_PARAM_ADVERTISING_ID)) {
+                   // Advertising ID of the device
+                }
+                if (data.containsKey(TenjinConsts.ATTR_PARAM_AD_NETWORK)) {
+                   // Ad network of the campaign 
+                }
+                if (data.containsKey(TenjinConsts.ATTR_PARAM_CAMPAIGN_ID)) {
+                   // Tenjin campaign ID  
+                }
+                if (data.containsKey(TenjinConsts.ATTR_PARAM_CAMPAIGN_NAME)) {
+                   // Tenjin campaign name
+                }
+            }
+        });
+
+        //Your other code...
+        ...
+
+    }
+```
+
+Below are the parameters, if available, that are returned to the attribution info callback:
```

Comment:
`Below are the parameters, if available, that are returned to the attribution info callback`
to
`Below are the attribution info parameters, that are returned when the attribution info callback is used`

write a note block mentioning they are returned if they are available.

Also an example on what is returned.